database/seeds/PermissionRoleTableSeeder.php

```diff
@@ -73,6 +73,12 @@ public function run()
                 'value'         => 1,
             ],
             [
+                'permission_id' => DB::table('permissions')->where('permission_name', '=', 'canStick')
```

Comment:
Not exactly linked to this PR, but we really need to clean up these migrations. There's way too much going on in them for my liking...